terraform/helm/genesis/values.yaml

```diff
@@ -69,6 +69,10 @@ genesis:
     internal_host_suffix: fullnode-lb
   # -- The local path for move modules in the docker image. Defaults to the aptos-framework in the aptoslabs/tools docker image
   moveModulesDir: /aptos-framework/move/modules
+  multicluster:
```

Comment:
same as other `values.yaml` -- add comment for experimental, and regenerate the docs